scout/adapter/mongo/base.py

```diff
@@ -120,5 +121,17 @@ def collections(self):
         """
         return self.db.list_collection_names()
 
+    def collection_stats(self, coll_name):
```

Comment:
Maybe I'm a bit biased now but it is really nice to use the `typing` types and utilise pythons type hints functionality. It becomes more easy to program since you get a lot for free with the editor (at least with PyCharm). One can also use [mypy](https://mypy.readthedocs.io/en/stable/) to check that input and output is correctly defined. So in this case: 
```suggestion
    def collection_stats(self, coll_name: str) -> dict:
```

Reply:
Good suggestion, I'll add the typing types!